Dockerfile

```
@@ -93,7 +93,7 @@ RUN set -eux; \
 	;
 
 # prevent the reinstallation of vendors at every changes in the source code
-COPY package.json yarn.lock ./
+COPY package.json yarn.* ./
```

```suggestion
COPY package.json yar[n].lock ./
```
is a safer option to avoid having errors for a non-existent file but it looks strange 😄